src/js/components/Tabs/doc.js

```diff
@@ -48,6 +48,28 @@ currently active tab changes.`,
 };
 
 export const themeDoc = {
+  'global.borderSize': {
+    description: 'The size of the border',
```

Comment:
can you please add a dot at the end of each `description`, it renders much nicely on the docs.